lib/winapi/winnt.h

```diff
@@ -283,8 +283,8 @@ typedef struct _XBE_SECTION_HEADER
 
 typedef struct _XBE_FILE_HEADER
 {
-    WORD Magic;
-    UCHAR Signature[256];
+    DWORD Magic;
+    UCHAR Signature[255];
```

Comment:
This is not correct, the signature is 256 bytes, not 255. You can compare with cxbe and xboxdevwiki.

Reply:
My testing suggested this was correct. Thrimbor tested better, showing the opposite and that my testing simply gave false positives.